src/rock/models/people/model.js

```diff
@@ -39,6 +39,16 @@ export class PhoneNumber extends Rock {
       code: 400, success: false, error: "Insufficient information",
     };
 
+    // make sure user doesn't already have a phone number on file
+    const query = {
+      where: {
```

Comment:
@richarddubay lets include the `Number` and we can remove the NumberTypeValueId. That way if the number they enter is on file, we can just hapily return success.

so
```js
const nonFormattedPhoneNumber = phoneNumber.replace(/[-+() ]/g, "");
const hasPhoneNumber = await PhoneNumberTable.findOne({ where: { PersonId: person.Id, Number: nonFormattedPhoneNumber } });
 if (hasPhoneNumber) return { code: 204, success: true };